packages/tools/pie-icons-react/icons/index.tsx

```diff
@@ -629,8 +629,8 @@ export { default as IconRssLarge } from './IconRssLarge';
 export { default as IconRss } from './IconRss';
 export { default as IconSaladLarge } from './IconSaladLarge';
 export { default as IconSalad } from './IconSalad';
-export { default as IconSandwichLarge } from './IconSandwichLarge';
-export { default as IconSandwich } from './IconSandwich';
+export { default as IconSandwhichLarge } from './IconSandwhichLarge';
```

Comment:
Looks like you need to pull in latest main, as this should be fixed (looks like a regression)